src/main/java/gregtech/integration/jei/GTOreInfo.java

```diff
@@ -96,6 +98,24 @@ public GTOreInfo(OreDepositDefinition definition) {
             //Group the output Ores
             groupedOutputsAsItemStacks = findUniqueBlocksAsItemStack(generatedBlocksAsItemStacks);
         }
+
+        // Generate list of weights for overlay text
+        List<FillerEntry> fillerEntries = blockFiller.getAllPossibleStates();
+        for (FillerEntry entries : fillerEntries) {
+            if (entries != null && !entries.getEntries().isEmpty()) {
+                for (Pair<Integer, FillerEntry> entry : entries.getEntries()) {
+                    totalWeight += entry.getKey();
+                }
+            }
+        }
+
+        FillerEntry entry = fillerEntries.get(0);
+        if (entry.getEntries() != null && !entry.getEntries().isEmpty()) {
```

Comment:
Could this just check `totalWeight != 0`?

Reply:
(I just copypasted stuff around with these loops)
`totalWeight` seems to take into account all `FillerEntry`s, while this and the other if statement only care about one at a time, so with multiple `FillerEntry`s being returned by `getAllPossibleStates`, it would not be equivalent